libobs/obs-encoder.h

```diff
@@ -120,6 +123,18 @@ struct obs_encoder_roi {
 	float priority;
 };
 
+struct gs_texture;
+
+/** Encoder input texture */
+struct encoder_texture {
+	/** Texture format and size */
+	struct video_scale_info info;
```

Comment:
This appears to be unused in all existing implementations and can probably be removed. Even if we add ARGB support we probably don't need this as all textures passed on an encoder must have the same format which should be negotiated on encoder init.